Easydict/NewApp/Utility/Extensions/QueryService+ConfigurableService/CustomOpenAIService+ConfigurableService.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  CustomOpenAIService+ConfigurableService.swift
+//  Easydict
+//
+//  Created by phlpsong on 2024/2/26.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Combine
+import Defaults
+import Foundation
+import SwiftUI
+
+// MARK: - CustomOpenAIService + ConfigurableService
+
+@available(macOS 13.0, *)
+extension CustomOpenAIService: ConfigurableService {
+    func configurationListItems() -> some View {
+        ServiceConfigurationSecretSectionView(
+            service: self,
+            observeKeys: [.customOpenAIAPIKey, .customOpenAIEndPoint]
```

Comment:
This should require input of the model before validation.

<img width="856" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/938a784e-f392-45c2-b9e1-67563c8d544e">

Reply:
fixed.